assignment/scripts/array-practice.js

```diff
@@ -75,20 +75,31 @@ console.log('The animals are now', animalArray);
 
 // 4.d TODO: Remove the food at the beginning of your array & 
 //     log both the food removed and the updated array
+removedFood= food.shift();
+console.log('Removed the first food', removedFood);
+console.log('The foods are now', food);
 
 // 4.e (STRETCH) TODO: Replace the second food in your array
 //      with another one of your favorite foods.
 //      Then log the updated array.
+food.splice(1, 1, 'burger');
+console.log(food);
 
 // 4.f (STRETCH) TODO: Sort your favoriteFoods array
 //     in reverse alphabetical order. Log the array.
+food.reverse();
+console.log(food);
 
 // 4.g (STRETCH) TODO: Convert your array to a string
 //     putting the word "and" between each item.
 //     eg "tacos and pizza and pasta". Log the string.
+joinFood = food.join(' and ');
+console.log(joinFood);
 
 // 4.h (STRETCH) TODO: Make a new array that combines 
 //     the favorite foods array with the animals array.
 //     Then log the new array.
 //     It should look something like:
-//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
\ No newline at end of file
+//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
+let combine = animals.concat(food);
```

Comment:
Could also use the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) for this.